cmd/influx/main.go

```diff
@@ -425,12 +425,9 @@ func (c *CommandLine) parseInto(stmt string) string {
 }
 
 func (c *CommandLine) Insert(stmt string) error {
-	stmt = stmt[7:]
-	if strings.EqualFold(stmt[:4], "into") {
-		stmt = c.parseInto(stmt[5:])
-	}
-	if c.RetentionPolicy == "" {
-		c.RetentionPolicy = "default"
+	stmt = strings.TrimSpace(strings.TrimSpace(stmt)[6:])
```

Comment:
The `[6:]` is meant to remove the `insert` at the beginning of the line. The assumption being that it's already there. Fair point about the magic numbers. I'll take them out.